backend/api/Program.cs

```diff
@@ -70,5 +71,16 @@ static IWebHostBuilder CreateWebHostBuilder(ProgramOptions options)
                 .UseUrls(config.GetValue<string>("ASPNETCORE_URLS"))
                 .UseStartup<Startup>();
         }
+
+        public static IHostBuilder CreateHostBuilder(string[] args)
```

Comment:
This is required by the scaffold command. The web host builder is ignored by the scaffold. If this is not present the connection string will not be pulled from the .env file correctly.